source/docs/articles/sites/code/hot-fixes.md

```diff
@@ -2,8 +2,10 @@
 title: Hot Fixes
 description: Learn how to deploy a hotfix on Pantheon.
 category:
-  - developing
+  - developing
 ---
+For Experts only. You should not need to attempt this if you use [Multidev](/docs/articles/sites/multidev) and keep commits from reaching Dev that you do not intend on deploying.

```

Comment:
Suggest adding something like language on *Using the Pantheon Workflow*:  "We do not recommend hotfixing. Hotfixes should be the exception, not the norm. Pushing a hotfix via Git is the only way to push code directly to Live without having to go through Dev and Test. Hotfixing is not a best practice."